modules/nf-core/wipertools/fastqgather/tests/main.nf.test

```diff
@@ -0,0 +1,72 @@
+nextflow_process {
+    name "Test Process WIPERTOOLS_FASTQGATHER"
+    script "../main.nf"
+    process "WIPERTOOLS_FASTQGATHER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "wipertools"
+    tag "wipertools/fastqgather"
+    tag "wipertools/fastqscatter"
+
+    setup {
+        run("WIPERTOOLS_FASTQSCATTER") {
+            script "../../fastqscatter/main.nf"
+            process {
+                """
+                input[0] = [
+                [ id:'pp_' ], // meta map
+                file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/fastq/test_1.fastq.gz", checkIfExists: true)
+                ]
+                input[1] = 2  // number of splits
+                """
+            }
+        }
+    }
+
+    test("merge two gzipped fastq - fastq.gz") {
+        config "./nextflow.config"
+
+        when {
+            params {
+                module_args = '--prefix pp_'
+            }
+            process {
+                """
+                input[0] = WIPERTOOLS_FASTQSCATTER.out.chunks.map { tuple( [id: 'fastq_chunks'], it[1] ) }
```

Comment:
FASTQCSCATTER already has a meta map, why are you replacing it here?

Reply:
yes, I know, thanks. I just wanted to change it

Reply:
Why don't you just give the meta map you want when you call it in the setup?

Reply:
just to remember me that I could change it later. I've edited